source/create-agg-pipeline.txt

```diff
@@ -403,260 +361,21 @@ your chosen view mode:
-
-To view your pipeline explain plan, click :guilabel:`Explain` in the 
-right corner of your pipeline toolbar.
-
-.. figure:: /images/compass/agg-explain-plan.png
-    :alt: Aggregation Pipeline Builder Explain Plan
-
-Your results are presented in this format: 
-
-.. figure:: /images/compass/agg-explain-results.png
-    :alt: Aggregation Pipeline Builder Explain Results
-
-.. _pipeline-custom-collation:
-
-Specify Custom Collation
-------------------------
-
-Use :manual:`custom collation </reference/collation/>` to specify
-language-specific rules for string comparison, such as rules for
-letter case and accent marks.
-
-To specify a custom collation:
-
-#. Click :guilabel:`More Options` in the top right corner of the pipeline 
-   builder. 
-
-   .. figure:: /images/compass/options-collation.png
-        :alt: Collation Option
-
-#. Enter your collation document.
-
-.. _export-agg-builder-results:
-
-Export Aggregation Results
---------------------------
-
-To export results from your aggregation pipeline:
-
-#. Connect to the deployment containing the
-   :doc:`collection </collections>` you wish to export data from.
-
-   To learn how to connect to a deployment, see
-   :ref:`connect-run-compass`.
-
-#. Navigate to your target collection.
-
-   You can either select the collection from the
-   :doc:`Collections </collections>` tab or click the collection 
-   in the left-hand pane. 
-
-#. Create an aggregation pipeline and run for results.
-
-   To learn how to create an aggregation pipeline, see :ref:`Create an 
-   Aggregation Pipeline <create-agg-pipeline>`. 
-
-#. Click :guilabel:`Export`.
-
-#. Choose the appropriate file type.
-
-   Under :guilabel:`Select Export File Type`, select either 
-   :guilabel:`JSON` or :guilabel:`CSV`. If you select :guilabel:`JSON`,
-   your data is exported to the target file as an array of JSON objects.
-
-#. Under :guilabel:`Output`, choose where to export the file to.
-
-   .. figure:: /images/compass/aggregation-export.png
-      :alt: Export Aggregation Data
-      :width: 70%
-
-#. Click :guilabel:`Export`. 
-
-
-.. _create-a-view:
-
-Create a View from Pipeline Results
------------------------------------
-   
-.. note::
-
-   Creating a view from pipeline results does not save the pipeline
-   itself.
-
-To create a view from your pipeline results:
-
-.. include:: /includes/fact-create-view-from-pipeline.rst
-
-.. _refresh-document-counts:
-
-Refresh Document Counts
------------------------
-
-When you delete or add a document, the :guilabel:`count results` value 
-on the :guilabel:`Aggregations` tab does not automatically update to 
-reflect the new document count.
-
-To update the :guilabel:`count results`:
-
-.. procedure::
-   :style: normal
-
-   .. step:: (Optional) If the results have not been counted, click 
-      the :guilabel:`count results` link.
-
-      .. figure:: /images/compass/agg-builder-click-count-results.png
-         :alt: Click the count results link
+Next Steps
```

Comment:
[suggestion] I'm not sure whether or not we need *both* a Next Steps section *and* a Learn More section. I'd be inclined to put them all under "Learn More." What do you think?